src/wallet/rpcwallet.cpp

```diff
@@ -3289,6 +3290,47 @@ void FundTransaction(CWallet& wallet, CMutableTransaction& tx, CAmount& fee_out,
         coinControl.fAllowWatchOnly = ParseIncludeWatchonly(NullUniValue, wallet);
     }
 
+    if (!solving_data.isNull()) {
+        if (solving_data.exists("pubkeys")) {
+            const UniValue pubkey_strs = solving_data["pubkeys"].get_array();
+            for (unsigned int i = 0; i < pubkey_strs.size(); ++i) {
+                const std::vector<unsigned char> data(ParseHex(pubkey_strs[i].get_str()));
```

Comment:
Check `IsHex` before calling `ParseHex` or use `ParseHexV`? (Here and below)

Reply:
Done